cmd/gobl/serve.go

```diff
@@ -88,8 +88,9 @@ func (s *serveOpts) version() echo.HandlerFunc {
 }
 
 type buildRequest struct {
-	Template json.RawMessage `json:"template"`
-	Data     json.RawMessage `json:"data"`
+	Template   json.RawMessage  `json:"template"`
+	Data       json.RawMessage  `json:"data"`
+	PrivateKey *dsig.PrivateKey `json:"privatekey"`
```

Comment:
I think I'd just set this to `Key` with an extra validation to ensure it's a private key.

Reply:
My only reason for calling it `privatekey` here was that with the Verify method, we'll need a `publickey`, and I wanted to make it clear that they're different keys.

WDYT?

Reply:
It is an API, so writing more is less of a concern. Let's leave it as `privatekey` then.